### DISCUSSION TITLE
Yolox_x in super-gradients

### DISCUSSION THREAD

**Discussion by yinmayoo185** [mikel-brostrom/yolo_tracking#1049] [Q&A]
In current version, I cannot try yolox_x. Some well pretrained yolo model for detection used yolox_x and I cannot try with yolox_x in current repo.

**Comment by mikel-brostrom** [comment]
Should be fairly easy. Notice that the SuperGradients repo is not super user friendly... Pun intended. A lot of things have to be set manually in order to adapt to you custom models. We cannot do much about this in this repo.

Substitute this line of code:

https://github.com/mikel-brostrom/yolo_tracking/blob/057435d17d6837b2e510e27503d925f99a0d1694/examples/detectors/yolonas.py#L39-L42

By:

```
yolo_nas = models.get(
    'yolox_x',
    checkpoint_path=<path/to/your/yolox-x/weights.pt>
).to(device) ```
